src/styles/index.scss

```
@@ -1,3 +1,141 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Helvetica", sans-serif;
+  color: #060b35;
+  margin: 0;
+  --secondary-color: #616070 ;
+  --blue-color: #00acdc;
+  --white-color: #fff;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  height: 408px;
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border: 1px #f3f3f3;
+
+  &__imag {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__product-code {
+    width: 98px;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 84px 16px 16px;
+    color: var(--secondary-color);
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    padding: 0 16px;
+  }
+
+  &__stars {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    height: 16px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 16px 16px;
+    padding: 0;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--secondary-color);
+    margin: 0;
+  }
+
+  &__product-price {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__buy {
+    width: 166px;
```

```suggestion
    width: 100%;
```
it should be width 100%.